src/data/drinkRecipes/saltyDog.ts

```diff
@@ -0,0 +1,51 @@
+import type { DrinkRecipe } from "types/DrinkRecipe"
+import { DrinkRecipeIngredientType } from "types/DrinkRecipe"
+
+const fakeImageUrl =
+  "https://www.theplunge.com/wp-content/uploads/2017/04/Salty-Dog-cocktail-cropped.png"
+
+const saltyDog: DrinkRecipe = {
+  name: "Salty Dog",
+  imageUrl: fakeImageUrl,
+  measuredIngredients: [
+    {
+      type: DrinkRecipeIngredientType.Measured,
+      ingredient: "Gin",
+      unit: "oz",
+      unitAmount: 1.5,
+      alternates: ["Vodka"],
+    },
+    {
+      type: DrinkRecipeIngredientType.Measured,
+      ingredient: "Grapefruit Juice",
+      unit: "oz",
+      unitAmount: 3.0,
+      alternates: [],
+    },
+  ],
+  numberedIngredients: [
+    {
+      type: DrinkRecipeIngredientType.Numbered,
+      ingredient: "Grapefruit",
+      suffix: "Slice",
+      amount: 1.0,
+      alternates: [],
+    },
```

Comment:
When the unit is 1 and the ingredient is a fruit, I usually put these in the portionless ingredients, with the suffix "Slice". The reason is it keeps the ingredient as "Grapefruit" and doesn't force us to add a new ingredient to the ingredientToIngredientAlias list, `"Grapefruit Slice": ["Grapefruit"]` which slows searching a bit. Adding suffix to the numbered ingredient types does get around that, but when the unit is 1, I don't think we need it as a numbered and can use the suffix that's already in the portionless ingredient type.

What do you think?